examples/tabsets/tabsets_quarto.qmd

```diff
@@ -0,0 +1,24 @@
+---
+title: "Tabsets"
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
```

Comment:
This is less of a review and more of a curiosity that since we don't have any other code chunks, we can safely remove this line, right?

Reply:
Yes, it is redundant here. This is how it ends when you copy-paste things ;)

Reply:
Got it! Thank you for confirming!